sdk/eventhub/azure-eventhub-checkpointstoretable/tests/test_storage_table_partition_manager.py

```diff
@@ -1,8 +1,119 @@
 import pytest
 from azure.eventhub.extensions.checkpointstoretable import TableCheckpointStore
+from azure.data.tables import TableServiceClient
+import uuid
+import warnings
+import random
+import os
 
+STORAGE_CONN_STR = os.environ['Azure-storage-conn']
```

Comment:
```suggestion
STORAGE_CONN_STR = os.environ.get("Azure-storage-conn", "<some_fake_value")
```

You need to provide a fake value for when running your tests in CI, this value should look very similar to your actual storage connection string but with the `key` portion replaced with something that is not a key (ie `fakestoragekey` is used a lot.